File: desh/ege2026kp/24solve/24-136.py
# Автор: В.Н. Шубинкин

with open('24-j5.txt') as tf:
    s = tf.read().strip()

print(s.count('OCK') - s.count('STOCK'))

count = 0
for i in range(len(s)-2):
  if s[i:i+3] == "OCK" and \
     (i < 2 or s[i-2:i] != "ST"):
    count += 1

print( count )

File: desh/ege2026kp/24solve/24-136.cpp
// Автор: М. Нехорошева

//Текстовый файл 24-s1.txt состоит не более чем из 106 заглавных латинских букв (A..Z). 
//Текст разбит на строки различной длины. Определите количество строк, 
//в которых буква J встречается чаще, чем буква E.
#include <iostream>
#include <fstream>
using namespace std;

int main() 
{
	fstream F;  //открываем файл в режиме чтения  
	F.open("24-s1.txt");
	string s;
	int k=0;
	for (;!F.eof();)
	{
		int k_J=0, k_E=0;
		getline(F,s);
		for (int j=0; j<s.size();++j)
		{
			if (s[j]=='J') ++k_J;
			if (s[j]=='E') ++k_E;
		}
		if (k_J>k_E) ++k;
	}
	cout <<k;

}

